DarkMagician/DarkMagician.Console/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿Console.ForegroundColor = ConsoleColor.Cyan;
+Console.WriteLine("WELCOM TO DARK WORLD! ");
+Console.WriteLine("PLACE FULL OF MONSTERS, MAGIC AND... HOT ANIMATED GIRLS");
+Console.WriteLine();
+Console.WriteLine("WHAT IS YOUR NAME?");
+Console.ForegroundColor = ConsoleColor.White;
+var name = Console.ReadLine();
+Console.ForegroundColor = ConsoleColor.Cyan;
+Console.WriteLine("SO..." + name + " WHATS YOUR CLASS? WARRIOR? ROGUE? OR MAGICIAN?");
+Console.ForegroundColor = ConsoleColor.White;
+var playerClass = Console.ReadLine();
+if (playerClass == "WARRIOR")
```

Comment:
w takiej sytuacji zamiast `ifów` warto spojrzeć na `switch`

Reply:
ale równie dobrze można używać ifa, switch to nie jest jakiś wymóg, kwestia gustu i sytuacji, najważniejsze żeby kod był czytelny i zrozumiały dla programistów